orc8r/cloud/go/services/certifier/obsidian/handlers/handlers.go

```diff
@@ -0,0 +1,208 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/labstack/echo"
+	"golang.org/x/crypto/bcrypt"
+
+	"magma/orc8r/cloud/go/obsidian"
+	"magma/orc8r/cloud/go/services/certifier"
+	certProtos "magma/orc8r/cloud/go/services/certifier/protos"
+	"magma/orc8r/cloud/go/services/certifier/storage"
+)
+
+const (
+	UserParam  = ":username"
+	User       = "user"
+	ListUser   = obsidian.V1Root + User
+	ManageUser = ListUser + obsidian.UrlSep + UserParam
+	Login      = ListUser + obsidian.UrlSep + "login"
+)
+
+func GetHandlers(storage storage.CertifierStorage) []obsidian.Handler {
```

Comment:
So we probably don't want the handlers to share the underlying store with the certifier grpc servicer, since it breaks the DAO abstraction. Instead, have the handlers be a stateless component that forwards requests to the certifier grpc servicer